examples/bond_port_up.yml

```diff
@@ -0,0 +1,27 @@
+---
+# eth1 --+-- bond0
+# eth2 --'
+interfaces:
+  - name: bond0
+    type: bond
+    state: up
+    link-aggregation:
+      mode: active-backup
+      options:
+        miimon: 140
+      port:
+        - eth1
+        - eth2
```

Comment:
We should not suggest user to define port names in two places.

And in commit message, we should explain an error will happen if `port` does not agree with `ports-config`.

Reply:
I revised the commit message and added the unit case to assert that an error will be raised.

Reply:
Maybe I need to add some validation function to raise specific errors.